ofrak_core/pytest_ofrak/mark.py

```diff
@@ -0,0 +1,41 @@
+import asyncio
+from typing import Mapping, Sequence, Type
+import os
+import pytest
+
+from ofrak.model.component_model import ComponentExternalTool
+from ofrak.component.abstract import AbstractComponent
+
+
+def skipif_windows():
+    return pytest.mark.skipif(os.name == "nt", reason="Test cannot run on Windows.")
+
+
+async def uninstalled_deps(component: Type[AbstractComponent]) -> Sequence[ComponentExternalTool]:
+    installed = await asyncio.gather(
+        *(dep.is_tool_installed() for dep in component.external_dependencies)
+    )
+    return [
+        dep for dep, installed in zip(component.external_dependencies, installed) if not installed
+    ]
+
+
+async def all_uninstalled_deps(
+    components: Sequence[Type[AbstractComponent]],
+) -> Mapping[Type[AbstractComponent], Sequence[ComponentExternalTool]]:
+    uninstalled = await asyncio.gather(*(uninstalled_deps(component) for component in components))
+    return {component: deps for component, deps in zip(components, uninstalled) if deps}
+
+
+def requires_deps_of(*args: Type[AbstractComponent]):
```

Comment:
Right now the docstring does a much better job of describing what this marker does: it skips tests if dependencies are not installed. Can you pick a name that more accurately describes this?

Reply:
renamed to `skipif_missing_deps` (and using pytest's custom marker mechanism)